build/Targets/Test.targets

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
   
   <ItemGroup Condition="'$(OutputType)' == 'Exe' ">
-    <Compile Include="$(MSBuildThisFileDirectory)TestProgram.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)TestProgram.cs" Visible="false" />
```

Comment:
I don't think this should be 'Visible=false'

It is nice to be able to view this file and open it from within VS.